data/locale/ru-RU.ini

```
@@ -0,0 +1,38 @@
+NDIPlugin.Default="По умолчанию"
+NDIPlugin.NDISourceName="NDI™ источник"
+NDIPlugin.SourceProps.SourceName="Названия источника"
+NDIPlugin.SourceProps.Bandwidth="Пропускная способность"
+NDIPlugin.SourceProps.Sync="Синхронизация"
+NDIPlugin.SourceProps.HWAccel="Разрешить аппаратное ускорение"
+NDIPlugin.SourceProps.AlphaBlendingFix="Fix alpha blending (adds a filter to this source)"
+NDIPlugin.SourceProps.ColorRange="YUV Range"
+NDIPlugin.SourceProps.ColorRange.Partial="Partial"
+NDIPlugin.SourceProps.ColorRange.Full="Full"
+NDIPlugin.SourceProps.ColorSpace="YUV Color Space"
+NDIPlugin.SourceProps.Latency="Режим задержки"
+NDIPlugin.SourceProps.Latency.Normal="Обычный (безопасно)"
+NDIPlugin.SourceProps.Latency.Low="Низкий (экспериментальный)"
+NDIPlugin.BWMode.Highest="Самый высокий"
+NDIPlugin.BWMode.Lowest="Низший"
+NDIPlugin.BWMode.AudioOnly="Только аудио"
+NDIPlugin.SyncMode.Internal="Внутренний"
+NDIPlugin.SyncMode.NDITimestamp="Сеть"
+NDIPlugin.SyncMode.NDISourceTimecode="Время источника"
+NDIPlugin.OutputName="NDI™ выход"
+NDIPlugin.OutputProps.NDIName="Название выхода"
+NDIPlugin.FilterProps.NDIName="Название NDI"
+NDIPlugin.FilterProps.NDIName.Default="Dedicated NDI Output"
+NDIPlugin.FilterProps.ApplySettings="Применить изменения"
+NDIPlugin.Menu.OutputSettings="Настройки выхода NDI™"
+NDIPlugin.OutputSettings.DialogTitle="Настройки выхода NDI™"
+NDIPlugin.OutputSettings.GroupBox.Main="Главный выход"
+NDIPlugin.OutputSettings.GroupBox.Preview="Предварительный выход"
+NDIPlugin.OutputSettings.Main.Name="Название главного выхода"
+NDIPlugin.OutputSettings.Preview.Name="Название предварительного выхода"
+NDIPlugin.FilterName="Выделенный выход NDI™"
+NDIPlugin.AudioFilterName="Выделенный выход NDI™ (Только аудио)"
+NDIPlugin.PremultipliedAlphaFilterName="obs-ndi - Fix alpha blending"
+NDIPlugin.LibError.Title="NDI™ Runtime не найден"
+NDIPlugin.LibError.Message.Win="NDI™ Runtime не найден.<br>Скачать установщик можно здесь: <a href='http://new.tk/NDIRedistV4'>http://new.tk/NDIRedistV4</a>"
```

Please use the new URL in these strings. I'll take care of uncoupling this from the localizations in the future.

```suggestion
NDIPlugin.LibError.Message.Win="NDI™ Runtime не найден.<br>Скачать установщик можно здесь: <a href='https://ndi.palakis.fr/runtime/ndi-runtime-4.5.1-Windows.exe'>https://ndi.palakis.fr/runtime/ndi-runtime-4.5.1-Windows.exe</a>"
```